qml/InitDSMonitorDialog.qml

```diff
@@ -72,31 +68,37 @@ Dialog {
             }
             TextField {
                 id: discoveryServerGuid
-                placeholderText: "44.53.00.5f.45.50.52.4f.53.49.4d.41"
+                placeholderText: "GUID example: 44.53.00.5f.45.50.52.4f.53.49.4d.41"
                 implicitWidth: dialogLayout.width
                 text: "44.53.00.5f.45.50.52.4f.53.49.4d.41"
                 selectByMouse: true
                 validator: RegExpValidator {
                     regExp: /^(([0-9a-fA-F][0-9a-fA-F]\.){11})([0-9a-fA-F][0-9a-fA-F])$/
```

Comment:
This regex is to strict and do not allow to modify the GUID, so the default one could not be changed (what it is very useful because the common ones are the one by default changing one or two chars).